src/expressions/array-expr.md

```diff
@@ -26,16 +26,16 @@ r[expr.array.array-behavior]
 This produces an array containing each of these values in the order they are written.
 
 r[expr.array.repeat]
-The syntax for the second form is two expressions separated by a semicolon (`;`) enclosed in square brackets.
+The syntax for the second form is an expression and a const argument separated by a semicolon (`;`) enclosed in square brackets.
```

Comment:
What's the theory behind "expression and a const argument"?

Reply:
as in, it consists of *both* an expression and a const argument. it is a bit ambiguous let me update that

Reply:
[expr; constargument]